code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -52,7 +52,8 @@
 	switch (severity)
 		if(1)
 			b_loss += 500
-			add_attacklogs(src, whodunnit, "got caught in an explosive blast from", addition = "Severity: [severity], Gibbed", admin_warn = TRUE)
+			if((src.key || src.ckey) && whodunnit)
```

Comment:
Checking either key or ckey is sufficient, no need to check both.
Also, isn't it better to check for a mind, instead? So that if an explosion hits my corpse while I'm ghosted it's still logged.

Reply:
Niche cases like scrying orbs, astral projection. I think it's worth checking for.

Reply:
Players without a mind could be just braindead (logged off) though, so I'll try use both at once.

Reply:
No, that's the point of a mind. It persists even if the mob no longer has a key associated to it.